docs/externalGrading.md

```diff
@@ -238,36 +238,67 @@ Running PrairieLearn locally with externally graded question support looks somet
+graded question:
 
 ```bash
 standard_init_linux.go:207: exec user process caused "no such file or directory"
 ```
 
-The error occurs during grading as a result of an OS new-line incompatibility with the `entrypoint` script in the externally
-graded question. One solution for this is to make a `.gitattributes` files in your PL repository with the line
+One solution for this is to make a `.gitattributes` files in your PL repository with the line
 `*.sh text eol=lf`. This tells the GitHub client to write the script files in native Linux
 format instead of converting them for Windows (which breaks mapping them back into docker). 
 This mimics the [`.gitattributes` file in the main PrairieLearn repo](https://github.com/PrairieLearn/PrairieLearn/blob/master/.gitattributes).
 
+###### `invalid mode: /grade`
+
+This error occurs when `HOST_JOBS_DIR` cannot be accessed:
+
+```sh
+error: Error processing external grading job 1
+error: handleGraderErrorUnable to launch Docker container for grading: (HTTP code 500) server error - invalid mode: /grade
+```
+
+1. Verify that the `pl_ag_jobs` directory was created successfully.
+2. Verify the following quirks about `HOST_JOBS_DIR`:
+    - Use Unix-style slashes even though you are using PowerShell (i.e., use `-e HOST_JOBS_DIR=/c/Users/Tim/pl_ag_jobs`, **not** `-e HOST_JOBS_DIR=C:\Users\Tim\pl_ag_jobs`).
+    - Spell out the full path without using `$HOME` (i.e., use `-e HOST_JOBS_DIR=/c/Users/Tim/pl_ag_jobs`, **not** `-e HOST_JOBS_DIR=$HOME/pl_ag_jobs`).
+3. Verify your Windows/Docker shared access:
+    _ Redo Docker's access to `C:` drive (or whichever drive your course directory is on) by right-clicking the Docker "whale" icon in the taskbar > clicking "Settings" > unchecking `C:` drive > re-checking `C:` drive.
+    - If still not working, restart Docker.
+    - If still not working, restart Windows.
+
 #### Running locally (native, not on Docker)
 
-When not running in Docker, things are easier. The Docker socket can be used normally, and we're able to store job files automatically. By default, they are stored in `$HOME/pl_ag_jobs` on Unix-based systems and `$USERPROFILE/pl_ag_jobs` on Windows. However, if you run PrairieLearn with a an environment variable `JOBS_DIR=/abs/path/to/my/custom/directory/`, that directory will be used instead. Note that this environment variable has no effect when running on Docker.
+When not running in Docker, things are easier. The Docker socket can be used normally, and we're able to store job files automatically without setting `HOST_JOBS_DIR`. By default, they are stored in `$HOME/.pljobs` on Unix-based systems and `$USERPROFILE/.pljobs` on Windows. However, if you run PrairieLearn with an environment variable `JOBS_DIR=/abs/path/to/my/custom/jobs/directory/`, that directory will be used instead. Note that this environment variable has no effect when running on Docker, in which case the jobs directory is specified using `HOST_JOBS_DIR` instead of `JOBS_DIR`.
```

Comment:
Also changed `pl_ag_jobs` to `.pljobs` (assuming I understood correctly [externalGraderLocal.js#L240](https://github.com/PrairieLearn/PrairieLearn/blob/master/lib/externalGraderLocal.js#L240))

Reply:
Perfect!